lua/lazyvim/plugins/extras/lang/helm.lua

```diff
@@ -0,0 +1,22 @@
+require("lazyvim.util.lsp").on_attach(function(client, buffer)
+  if client.name == "docker_compose_language_service" then
+    if vim.api.nvim_buf_get_option(buffer, "filetype") == "helm" then
```

Comment:
The `require("lazyvim.util.lsp").on_attach` is deprecated. I think it's better to use this instead.

```lua
require("lazyvim.util.lsp").lsp.on_attach
```

Also, the `vim.api.nvim_buf_get_option` is deprecated. My take is to use this.

```lua
vim.api.nvim_get_option_value("filetype", { buf = buffer })
```

Reply:
When I use `require("lazyvim.util.lsp").lsp.on_attach`, I get an error 
```
undefined field `lsp`
```

Did you mean `require("lazyvim.util").lsp.on_attach`?
Isn't this the same as `require("lazyvim.util.lsp").on_attach`?

Reply:
Whoops, my bad. You are certainly correct.